src/repr/src/optimize.rs

```diff
@@ -119,6 +119,7 @@ optimizer_feature_flags!({
     reoptimize_imported_views: bool,
     // Enables the value window function fusion optimization.
     enable_value_window_function_fusion: bool,
+    enable_reduce_unnest_list_fusion: bool,
```

Comment:
Documentation!

Reply:
Sorry, I was thinking to stop adding docs for these optimizer flags, because most of these (except for `reoptimize_imported_views`) are simply the same thing as the feature flag of the same name.

Now I've added a comment that explicitly points to the feature flag of the same name.